cmd/commands/runtime.go

```diff
@@ -1554,6 +1568,21 @@ func postInstallationHandler(ctx context.Context, opts *RuntimeInstallOptions, e
 	printSummaryToUser()
 }
 
+func HandleCliStep(ctx context.Context, r reporter.AnalyticsReporter, event reporter.CliEventType, message string, err error) {
```

Comment:
This should be a private function inside this package. not public.

Reply:
also `reporter` should not be passed here (use `.Get()`).

Reply:
also `ctx` is not used here

Reply:
also you can change this to use the `...error` as last parameter, then it won't be required to pass `nil` if there is no error.